plugins/importexport/native/filter/NativeXmlIssueFilter.php

```diff
@@ -337,7 +337,7 @@ public function parseSection($node)
         $section->setContextId($context->getId());
         $reviewFormId = $node->getAttribute('review_form_id');
         $section->setReviewFormId($reviewFormId ? (int) $reviewFormId : null);
-        $section->setSequence($node->getAttribute('seq'));
+        $section->setSequence((int) $node->getAttribute('seq'));
```

Comment:
This should be ok, but as the field is a float, it's better to use another typecast `(float)`.